app/lib/theme/widgets/ReadOnlyAdjustableCalculatedValueWidget.tsx

```diff
@@ -22,11 +22,11 @@ const ReadOnlyAdjustableCalculatedValueWidget: React.FC<WidgetProps> = (
           {
             <NumberFormat
               thousandSeparator
-              fixedDecimalScale={isMoney || isPercent}
+              fixedDecimalScale={isMoney || isPercentage}
               id={id}
               prefix={isMoney ? "$" : ""}
-              suffix={isPercent ? "%" : ""}
-              decimalScale={isMoney || isPercent ? 2 : 10}
+              suffix={isPercentage ? "%" : ""}
+              decimalScale={isMoney || isPercentage ? 2 : 10}
```

Comment:
We can probably add support for configurable decimal places from the uiSchema in a further refactor

Reply:
I mostly already added it; I'll add it quickly here